Sources/BraintreePayPal/BTPayPalCheckoutRequest.swift

```diff
@@ -56,14 +56,14 @@ import BraintreeCore
 }
 
 /// Options for the PayPal Checkout flow.
-@objcMembers public class BTPayPalCheckoutRequest: BTPayPalRequest {
+@objcMembers open class BTPayPalCheckoutRequest: BTPayPalRequest {
 
     // MARK: - Public Properties
 
     ///  Used for a one-time payment.
     ///
     ///  Amount must be greater than or equal to zero, may optionally contain exactly 2 decimal places separated by '.' and is limited to 7 digits before the decimal point.
-    public let amount: String
+    public var amount: String
```

Comment:
Changed to a var because we now mutate this in the `super.init` of `BTPayPalNativeCheckoutRequest`